examples/arkode/C_serial/ark_kpr_mri.c

```diff
@@ -151,9 +162,14 @@ int main(int argc, char* argv[])
   SUNLinearSolver LSf               = NULL; /* fast linear solver object    */
   SUNMatrix As                      = NULL; /* matrix for slow solver       */
   SUNLinearSolver LSs               = NULL; /* slow linear solver object    */
-  sunbooleantype implicit_slow;
-  sunbooleantype imex_slow = SUNFALSE;
-  sunbooleantype deduce    = SUNFALSE;
+  sunbooleantype implicit_slow      = SUNFALSE;
+  sunbooleantype imex_slow          = SUNFALSE;
+  sunbooleantype explicit_slow      = SUNFALSE;
+  sunbooleantype no_slow            = SUNFALSE;
+  sunbooleantype implicit_fast      = SUNFALSE;
+  sunbooleantype explicit_fast      = SUNFALSE;
+  sunbooleantype no_fast            = SUNFALSE;
+  sunbooleantype deduce             = SUNFALSE;
```

Comment:
`deduce` what? can we make the variable name a little more descriptive (I realize it has the same name as before).